example/gluon/wavenet/main.py

```diff
@@ -0,0 +1,60 @@
+"""
+Descrition : main module to run code
+"""
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import argparse
+from trainer import Train
+
+def main():
+    """
+    Description : run wavenet code using argument info
+    - batch_size : Define batch size (default=64)
+    - epochs : Define the total number of epochs (default=1000)
+    - mu : Define mu value for mu-law algorithm (default=128)
+    - n_residue : Define number of residue (default=24)
+    - n_skip : Define number of skip (default=128)
+    - dilation_depth : Define dilation depth (default=10)
+    - use_gpu : use gpu for trainin (default=True)
+    - generation : whether or not to generate a wave file for model (default=True)
+    - load_file : file name in loading wave file
```

Comment:
mention default value here?

Reply:
add default value

Reply:
please mention the default value here as well

Reply:
could you add the default value in this comment as well

Reply:
Update default value #f8bd03e